src/pages/app-development/starter-kit/webhook-ingestion.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Webhook ingestion
+description: Learn how to ingest webhooks with Adobe Commerce Extensibility Starter Kit.
+keywords:
+ - Extensibility
+ - App Builder
+ - API Mesh
+ - Events
+ - REST
+ - Tools
+---
+
+# Webhooks ingestion
```

Comment:
Maybe this has to be the other way around: `Ingestion webhook`: it is a webhook that allows the ingestion of events